.github/workflows/publish-package.yml

```
@@ -0,0 +1,31 @@
+name: Publish to Github Packages
+on:
+  push:
+    branches:
+      - main
```

`pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```